manifests/init.pp

```diff
@@ -132,6 +134,7 @@
   $cassandra_yaml_tmpl          = 'cassandra/cassandra.yaml.erb',
   $commitlog_directory          = undef,
   $commitlog_directory_mode     = '0750',
+  $manage_config_file           = true,
```

Comment:
Can you add the Boolean datatype to this?

Reply:
So, in the format: `Boolean $manage_config_file           = true,`?

There are other Boolean parameters for this class. Should I update all the boolean parameters to have their type defined? Should I update all the parameters to have their data type defined?

Just trying to understand if you have some balance of consistency you are trying to maintain, or just attempting this as a start a better style pattern.

Reply:
ah, sorry. I should be more clear. Puppet supports datatypes since a few releases. Migrating all existing parameter in all modules (we've 140 at the moment) takes quite some time. We would like to have datatypes at least for new parameters. It would be really awesome if you're interested in adding datatypes to other params as well, but that should happen in another pull request. The syntax you mentioned is correct. We've our review guidelines at the bottom of this page: https://voxpupuli.org/docs/#reviewing-a-module